package dev.vality.repairer.exception;

public class DaoException extends RuntimeException {

    public DaoException(Throwable cause) {
        super(cause);
    }

}
